app/Http/Kernel.php

```diff
@@ -65,5 +65,6 @@ class Kernel extends HttpKernel
         'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
         'checkAdmin' => \App\Http\Middleware\CheckAdmin::class,
+        'lang' => \App\Http\Middleware\Localization::class,
```

Comment:
middleware Locale apply to tất cả các route của web thìbỏ luôn vào route group của web